stage1/init/kvm.go

```diff
@@ -34,12 +34,12 @@ import (
 )
 
 // KvmNetworkingToSystemd generates systemd unit files for a pod according to network configuration
-func KvmNetworkingToSystemd(p *stage1commontypes.Pod, n *networking.Networking) error {
+func KvmNetworkingToSystemd(p *stage1commontypes.Pod, n *networking.Networking, debug bool) error {
```

Comment:
This should have access to the debug variable because kvm.go and init.go are in the same package ("main").